lib/device-sockets/ios/socket-proxy-factory.ts

```diff
@@ -119,6 +119,7 @@ export class SocketProxyFactory implements ISocketProxyFactory {
 
 		let socketFileLocation = temp.path({ suffix: ".sock" });
 		server.listen(socketFileLocation);
+		this.$logger.info("socket-file-location: " + socketFileLocation);
```

Comment:
as far as I understand, we need this only when `this.$options.client` is false (`--no-client` is passed)
When started from CLI, the --no-client will not be passed and users should not see this row?

If my understanding is correct, can you add the if, something like:
```TypeScript
if(!this.$options.client)  {
    this.$logger.info("socket-file-location: " + socketFileLocation);
}
```
and in the constructor:
```TypeScript
constructor(private $logger: ILogger,
 		private $projectData: IProjectData,
 		private $projectDataService: IProjectDataService,
 		private $options: IOptions) { }
```